theseus/core/cost_function.py

```diff
@@ -11,7 +11,11 @@
 
 import torch
 import torch.autograd.functional as autogradF
-from functorch import jacrev, vmap
+
+try:
+    from torch.func import jacrev, vmap
+except ModuleNotFoundError:
+    from functorch import jacrev, vmap  # type: ignore
```

Comment:
Is this in case torch < 2.0 is installed and functorch is separate install+import?

Reply:
Yes!